datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -52,51 +57,69 @@ pub const DEFAULT_PARQUET_EXTENSION: &str = ".parquet";
 /// The Apache Parquet `FileFormat` implementation
 #[derive(Debug)]
 pub struct ParquetFormat {
-    enable_pruning: bool,
+    // Session level configuration
+    config_options: Arc<RwLock<ConfigOptions>>,
+    // local overides
+    enable_pruning: Option<bool>,
```

Comment:
By changing these to `Option` I think it is now clearer that if they are left at default, the (documented) value from ConfigOptions is used